psi4/src/psi4/libmints/matrix.cc

```diff
@@ -1251,7 +1251,7 @@ double Matrix::rms() {
     double sum = (double)0.0;
     long terms = 0;
     for (int h = 0; h < nirrep_; ++h) {
-#pragma omp parallel for reduction(+ : sum)
+#pragma omp parallel for reduction(+ : sum, terms)
         for (int i = 0; i < rowspi_[h]; ++i) {
             for (int j = 0; j < colspi_[h ^ symmetry_]; ++j) {
                 sum += matrix_[h][i][j] * matrix_[h][i][j];
```

Comment:
Might be better to write the following above the OMP loop, but either way fixes the error.

```
terms += rowspi_[h] * colspi_[h ^ symmetry_];
```

Reply:
Fair enough. I was also wondering about the loop below: the `absmax()` function can also be parallelised the same way (using custom resolution in OMP >=4.5). Which versions of OMP do we target?

Reply:
I don't think we target any version of OMP at the moment, when 5.0 comes out I think we will have to review that strategy. @loriab may have more comments.

Reply:
* Prior to the news that MSVC only supports OpenMP v2, I was scarcely aware of OpenMP versioning
* conda
  * linux: conda `gcc` `v7.3`, OpenMP is `v4.5`
  * linux: `icpc` `v2018.3` atop conda `gcc` `v7.3`, OpenMP is `v5.0`
  * mac: conda `clang` `v4.0.1`, OpenMP is `v3.1` whether `libomp` _or_ `libiomp5` (suggesting to me that the compiler is the constraint)
  * mac: `icpc` `v2018.2` atop conda clang `v4.0.1`, looks like I need to renew my license :-(